docs/getting-started/linux-manual.md

```diff
@@ -160,11 +162,29 @@ $ docker run --rm -e SPLUNK_ACCESS_TOKEN=12345 -e SPLUNK_REALM=us0 \
 
 > Use of a SemVer tag over `latest` is highly recommended.
 
-In the case of Docker, a volume mount may be required to load custom
-configuration as shown above.
+In the case of Docker, a volume mount may be required to store the custom
+configuration file as shown above.
 
 If the custom configuration includes a `memory_limiter` processor then the
 `ballast_size_mib` parameter should be the same as the
 `SPLUNK_BALLAST_SIZE_MIB` environment variable. See
 [gateway_config.yaml](../../cmd/otelcol/config/collector/gateway_config.yaml)
 as an example.
+
+Using `SPLUNK_CONFIG_YAML` instead of `SPLUNK_CONFIG` would be something like:
+
+```bash
+$ SPLUNK_CONFIG_YAML="extensions:\n  health_check:\n    endpoint: 0.0.0.0:13133"
+
+$ docker run --rm -e SPLUNK_ACCESS_TOKEN=12345 -e SPLUNK_REALM=us0 \
+    -e SPLUNK_CONFIG_YAML=${SPLUNK_CONFIG_YAML} -p 13133:13133 -p 14250:14250 \
+    -p 14268:14268 -p 4317:4317 -p 6060:6060 -p 7276:7276 -p 8888:8888 \
+    -p 9080:9080 -p 9411:9411 -p 9943:9943 \
+    --name otelcol quay.io/signalfx/splunk-otel-collector:latest
+```
+Where `SPLUNK_CONFIG_YAML` is the one-line version of the multi-line configuration YAML below:
+```yaml
+extensions:
+  health_check:
+     endpoint: 0.0.0.0:13133
```

Comment:
Is this complete?  A service definition is required to load the extension right?

Reply:
No, it's not complete. It's just an example showing some yaml. Do you think people will confuse that for an actual collector config?

Reply:
I'll add a note that the yaml is not a valid

Reply:
imo adding the basic service definition is adequate and doesn't leave an exercise for the reader